java/src/main/java/ai/rapids/cudf/HostColumnVectorCore.java

```diff
@@ -518,7 +519,7 @@ private Object readValue(int rowIndex) {
       case INT16: return offHeap.data.getShort(rowOffset);
       case BOOL8: return offHeap.data.getBoolean(rowOffset);
       case STRING: return getString(rowIndex);
-      default: throw new UnsupportedOperationException("Do not support " + type);
```

Comment:
We should at least have a follow on issue to update this API to return something for the various Decimal types.